src/status_im/navigation/events.cljs

```diff
@@ -30,8 +30,9 @@
 (rf/defn navigate-to-within-stack
   {:events [:navigate-to-within-stack]}
   [{:keys [db]} comp-id screen-params]
-  {:db (all-screens-params db (first comp-id) screen-params)
-   :fx [[:navigate-to-within-stack (conj comp-id (:theme db))]]})
+  {:db         (all-screens-params db (first comp-id) screen-params)
+   :dispatch-n [[:hide-bottom-sheet]]
```

Comment:
Bottom sheets should be hidden when navigate to another screen. Copied from `:navigate-to` event above in the code.